ckanext/unaids/react/components/BulkFileUploaderComponent/src/ModalBody.js

```diff
@@ -186,6 +186,7 @@ export default function ModalBody({
             .filter(x => x).length === 0;
         const enableButton = ![
             thereArePendingFiles,
+            !uploadInProgress,
```

Comment:
```        
const enableButton = ![
            thereArePendingFiles,
            !uploadInProgress,
            noDuplicateUploadActionsSet
        ].includes(false);
```
This conditional is quite complicated. You could simplify it a bit with:
```
const enableButton = [
            !thereArePendingFiles,
            uploadInProgress,
            !noDuplicateUploadActionsSet
        ].every(Boolean);
```
More info about `Array.every(Boolean)` https://stackoverflow.com/questions/53897673/check-if-all-values-in-array-are-true-then-return-a-true-boolean-statement-jav

This new aproach is much easier to read in English and understand when exactly we want the button to be disabled.
In other words we're chaning the logic expression of `!(!A or !B or !C)` into `(A and B and C)`.

Reply:
Yeah, this is good! I didn't know of `.every()`